src/python/WMCore/WorkQueue/WorkQueueReqMgrInterface.py

```diff
@@ -251,11 +249,16 @@ def reportRequestStatus(self, request, status, message = None):
         """
         if message:
             self.sendMessage(request, str(message), 'info')
```

Comment:
sendMessage actually log to logDB it is wrapper function for reqmgr1 and reqmgr2 common calls.